docs/book/src/cronjob-tutorial/testdata/project/test/utils/utils.go

```diff
@@ -54,10 +54,10 @@ func Run(cmd *exec.Cmd) ([]byte, error) {
 	_, _ = fmt.Fprintf(GinkgoWriter, "running: %s\n", command)
 	output, err := cmd.CombinedOutput()
 	if err != nil {
-		return output, fmt.Errorf("%s failed with error: (%v) %s", command, err, string(output))
+		return "", fmt.Errorf("%s failed with error: (%v) %s", command, err, string(output))
```

Comment:
Then, we are no longer return the output when we we should here.
We need return the output.

Reply:
The output is in the error message, though.  Generally the return values are zero-valued, _or_ the error is nil.  This is what Gomega checks, and so I replaced it with "" to adhere to the go idioms.